package.json

```diff
@@ -37,9 +37,9 @@
     "chai-as-promised": "^5.3.0",
     "devtron": "^1.3.0",
     "electron-builder": "5.2.1",
-    "electron-connect": "~0.3.9",
-    "electron-packager": "^7.0.1",
-    "electron-prebuilt": "1.2.8",
+    "electron-connect": "~0.6.0",
+    "electron-packager": "^8.0.0",
```

Comment:
You don't have to bump `electron-packager`. I'm working on updating build commands.
(In addition, `gulpfile.js` uses the deprecated parameter, `version-string`.)